src/js/builtins/ReadableStream.ts

```diff
@@ -168,18 +176,22 @@ export function readableStreamToFormData(
   stream: ReadableStream<ArrayBuffer>,
   contentType: string | ArrayBuffer | ArrayBufferView,
 ): Promise<FormData> {
-  return Bun.readableStreamToBlob(stream).then(blob => {
-    return FormData.from(blob, contentType);
-  });
+  if (!$isReadableStream(stream)) throw new TypeError("First argument must be a ReadableStream");
+
+  return Bun.readableStreamToBlob(stream).then(blob => FormData.from(blob, contentType));
```

Comment:
What was the purpose of this?

Reply:
Misread as an internal function call. Reverted.